source/features/highlight-deleted-and-added-files-in-diffs.tsx

```diff
@@ -14,7 +14,7 @@ async function loadDeferred(jumpList: Element): Promise<void> {
 	const retrier = setInterval(() => {
 		jumpList.parentElement!.dispatchEvent(new MouseEvent('mouseover'));
 	}, 100);
-	await observeOneMutation(jumpList);
+	await oneMutation(jumpList, {childList: true, subTree: true});
```

Comment:
I made the tough decision to avoid having this as a default because the options object now also includes a `filter` function.

I guess the alternative would be to support an _optional_ argument between the 2, but it seems awkward nowadays:

```js
// Both would work:
oneMutation(jumpList, {childList: true});
oneMutation(jumpList, filterFunction, {childList: true});
```

---

Anyway, `subTree` might not be required anywhere in this PR, but dropping it requires testing a few already-complex features.

Reply:
> Anyway, subTree might not be required anywhere in this PR, but dropping it requires testing a few already-complex features.

Maybe add a todo comment so we don't forget?